app/population/view/dashboard.njk

```diff
@@ -0,0 +1,22 @@
+{% extends "layouts/collection.njk" %}
+
+{% block pageContent %}
+  {% if resultsAsPopulationTable | length  %}
+    <div class="govuk-grid-row">
+    <section class="govuk-grid-column-full govuk-!-margin-bottom-6">
```

Comment:
Very minor, but this indentation still seems a bit off around `div > section` and `h2 > span`. It should be:

```nunjucks
{% extends "layouts/collection.njk" %}

{% block pageContent %}
  {% if resultsAsPopulationTable | length  %}
    <div class="govuk-grid-row">
      <section class="govuk-grid-column-full govuk-!-margin-bottom-6">
        <h2 class="govuk-heading-m govuk-!-margin-bottom-1">
          {{ t("dashboard::population.heading") }}

          <span class="govuk-!-display-block">
            {{ currentWeek | formatDateRangeWithRelativeWeek }}
          </span>
        </h2>

        {{ govukTable(resultsAsPopulationTable) }}
      </section>
    </div>
  {% endif %}
{% endblock %}
```

Reply:
Good catch! Fixed :)